students/asotelo/lesson03/strformat_lab.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/python3
+'''
+Author: Alex Sotelo
+Exercise 3.3
+Python 3 required
+Requirement: https://uwpce-pythoncert.github.io/PythonCertDevel/exercises/string_formatting.html
+'''
+
+#Task One
+fString = ( 2, 123.4567, 10000, 12345.67)
+element1 = int(fString[0])
+element2 = float(fString[1])
+element3 = int(fString[2])
+element4 = float(fString[3])
+
+print(f"'file_{element1:03d} :   {element2:.2f}, {element3:.2e}, {element4:.2e}'")
+
+#Task Two
+print("'file_{:03d} :   {:.2f}, {:.2e}, {:.2e}'".format(*fString))
+
+#Task Three
+in_tuple = ()
+
+def formatter(in_tuple):
+    numbers = len(in_tuple)
+    make_a_format_string = (f"the {numbers} are: ") + ", ".join(["{:d}"]*numbers)
+    return(make_a_format_string).format(*in_tuple)
+
+formatter(in_tuple)
+
+#Task Four
+og = ( 4, 30, 2017, 2, 27)
+
+def sFormat(og):
```

Comment:
Function names should be lowercase - if two words then separated by an underscore